packages/button/src/css/index.js

```diff
@@ -44,39 +44,15 @@ export default {
     border: '1px solid transparent'
   },
   '.psds-button:focus:before': {
-    content: ' ',
-    position: 'absolute',
-    top: '-1px',
-    left: '-1px',
-    right: '-1px',
-    bottom: '-1px',
-    background: core.colors.black,
-    zIndex: '-1',
-    borderRadius: '2px'
-  },
-  [`.psds-button.psds-theme--${themeNames.light}:focus:before`]: {
-    top: '-2px',
-    left: '-2px',
-    right: '-2px',
-    bottom: '-2px',
-    background: core.colors.bone
-  },
-  '.psds-button:focus:after': {
     content: ' ',
     position: 'absolute',
     top: '-4px',
     left: '-4px',
     right: '-4px',
     bottom: '-4px',
-    zIndex: '-2',
-    borderRadius: '4px',
-    background: core.colors.blue
-  },
-  [`.psds-button.psds-theme--${themeNames.light}:focus:after`]: {
-    top: '-5px',
-    left: '-5px',
-    right: '-5px',
-    bottom: '-5px'
+    zIndex: '-1',
```

Comment:
Is z-index: -1 required now?  I used it in the past to hide the bg of the pseudo element *behind* the button.  But now it's just a border.  If not z-index: -1, maybe need pointer-events: none?

Reply:
makes sense. i'll re-address after we go over the requirements